MC_Pound.cmd

```diff
@@ -95,7 +95,7 @@ small.ingot:
 
 ingot.grab:
 	 pause 1
-	 gosub STOW_RIGHT
+	 if $MC_TOOLBELT_FORGING != NULL then gosub STOW_RIGHT else gosub PUT_IT my $MC_HAMMER in my %tool.storage

```

Comment:
I use a toolbelt, and this fixed a problem I was having... However... the inline else is probably NOT going to work.